tools/dataform_assertions/includes/custom_test.js

```diff
@@ -0,0 +1,147 @@
+/*
+Numerical Data Assertions
+*/
+
+/*
+Telephone number checks
```

Comment:
rename custom_test.js file to something specific to phone numbers (e.g. phone_test.js)